open-metadata-resources/open-metadata-labs/egeria-server-config.ipynb

```diff
@@ -223,7 +223,7 @@
     "| discovery-engine     |   Yes    |   No     |   Yes    |   No     |   No     |   Yes    |   Yes    |     No      |\n",
     "| stewardship-action   |   Yes    |   No     |   Yes    |   No     |   No     |   Yes    |   Yes    |     No      |\n",
     "| -------------------- |  ------  |  ------  |  ------  |  ------  |  ------  |  ------  |  ------  | ----------  |\n",
-    "| data-engine          |   Yes    |   No     |   No     |   No     |   No     |   Yes    |   No     |     Yes     |\n",
+    "| data-engine          |   Yes    |   No     |   No     |   Yes    |   No     |   Yes    |   No     |     Yes     |\n",
```

Comment:
We haven't used this in the tutorial yet. Not in itself a big issue.
But I do think this is the wrong server -- cocoMDS4 is for users, not for operational engines to use. That should be cocoMDS1 I think